dependencies/config/config.go

```diff
@@ -40,6 +45,13 @@ func init() {
 	}
 }
 
+func ContainsSeparator(configEnv string) error {
```

Comment:
I want to test config, socket and container in case when user add namespace and for that I must transfer strings.Contains(someString, env.Separator) logic outside init method as an independent function. And this is the reason why I create ContainsSeparator method. 

Yep, I know it would be much clearer with doing something like: 
```
func ContainsSeparator(configEnv string) bool {
    if strings.Contains(configEnv, env.Separator) {
        fmt.Errorf("Config doesn't accept namespace")
        return true
    }
    return false
}
```
and test it like: 
```
It ("fails on trying to resolve a config path with namespace", func() {
        contains = ContainsSeparator(failingNamespaceConfig)
        Expect(contains).To(Equal(true))
    })
```
but... error message and just exit a program when user give you incorrect input and nil returns is more comfy and pretty solution.